gtdynamics/dynamics/Chain.cpp

```diff
@@ -0,0 +1,116 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2021, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file Chain.cpp
+ * @brief Create a serial kinematic chain.
+ * @author Dan Barladeanu, Frank Dellaert.
+ */
+
+#include <gtdynamics/dynamics/Chain.h>
+
+namespace gtdynamics {
+
+void Chain::monoidOperation(const Pose3 &aTb, const Matrix &bAj,
```

Comment:
Actually, could we just overwrite operator* ? Chain * Chain -> Chain

Reply:
done